pkgs/by-name/cr/create-dmg/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "create-dmg";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "create-dmg";
+    repo = "create-dmg";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-oWrQT9nuFcJRwwXd5q4IqhG7M77aaazBG0+JSHAzPvw=";
+  };
+
+  dontBuild = true;
+
+  installFlags = [ "prefix=$(out)" ];
+
+  meta = {
+    description = "A shell script to build fancy DMGs";
```

Please remove the indefinite article from the beginning of the description as recommended in [section "Meta attributes" in `pkgs/README.md`](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes)
```suggestion
    description = "Shell script to build fancy DMGs";
```

---

Added on most recent commit, thanks!

---

@afh I can confirm that change. Can we get an approval?

---

The changes look good to me, yet having all checks pass would be helpful, @sarahec.